recipe/meta.yaml

```diff
@@ -12,9 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
```

Comment:
Seems like to enable `cython` you need:
```
  script_env:
    - FONTTOOLS_WITH_CYTHON=1
```

See https://github.com/fonttools/fonttools/blob/705acc994f9f8785a582e981a4913394dc7699c0/tox.ini#L8C9-L8C32

Reply:
Added the environment variable successfully